lighthouse/start/beacon.sh

```diff
@@ -10,4 +10,5 @@
   --http-address 0.0.0.0 \
   --metrics \
   --metrics-address 0.0.0.0 \
-  --disable-upnp
\ No newline at end of file
+  --disable-upnp
```

Comment:
there needs to be a `\`behind the parameter, read more about this here: https://superuser.com/questions/508507/linux-bash-script-single-command-but-multiple-lines

Reply:
oh my bad, got it